src/main/java/com/firebolt/jdbc/connection/settings/FireboltProperties.java

```diff
@@ -176,7 +176,7 @@ private static String getDatabase(Properties properties, String path) throws Ill
 				return null;
 			} else {
 				Matcher m = DB_PATH_PATTERN.matcher(path);
-				if (m.matches()) {
+				if (m.find()) {
```

Comment:
As far as I remember there was a problem here. In one case DB name arrives with a prefix (/). But I am not 100% sure.

Reply:
Fixed. The problem was that in v1 the pattern was `/([a-zA-Z0-9_*\-]+)` because DB name is extracted from JDBC URL and it arrives here as "path" with slash prefix. The slash has been removed in new identity branch as not relevant because in this branch DB name is not prepended with slash. When I started merging both braches together I experienced this prollem with v1 URLs and instead of adding slash to pattern (that is right solution) replaced `matches()` by `find()` (that is incorrect solution). 

So, now the pattern looks like the following `/?([a-zA-Z0-9_*\-]+)` - it satisfies both cases and `find()` is changed back to `match()`